### PR TITLE
Add $idle store

### PR DESCRIPTION
**Pull request by a-polunin**
Add `$idle` store which is similar to `$pending`, `$failed` or` $succeeded`. That store shows if the fetching started or not.

### CODE REVIEW

**Review by igorkamyshev — APPROVED** [review]
Sounds great 🤗

Shall we add some tests for this new store?

**Comment by igorkamyshev** [comment]
(it's okay if you think it is not necessary, just let me know)

File: apps/website/docs/api/primitives/mutation.md [hunk 1]
```diff
@@ -9,9 +9,10 @@ const mutation: Mutation<Params, Data, Error>;
 
 // Stores
 mutation.$status; // Store<'initial' | 'pending' | 'done' | 'fail'>
+mutation.$idle; // Store<boolean>
```
Comment (igorkamyshev) [inline apps/website/docs/api/primitives/mutation.md:12]: 🤓 let's add a note about first appearance?

File: apps/website/docs/api/primitives/query.md [hunk 2]
```diff
@@ -12,6 +12,7 @@ const query: Query<Params, Data, Error, InitialData>; // InitialData is allowed
 query.$data; // Store<Data | InitialData>
 query.$error; // Store<Error | null>
 query.$status; // Store<'initial' | 'pending' | 'done' | 'fail'>
+query.$idle; // Store<boolean>
```
Comment (igorkamyshev) [inline apps/website/docs/api/primitives/query.md:15]: 🤓 let's add a note about first appearance? Like `// Store<boolean>, since v0.8.0`

**Comment by a-polunin** [comment]
> Sounds great 🤗
> 
> Shall we add some tests for this new store?

Oh, my bad, forgot about tests 😃 
Is the `create_headless_query.test.ts` a right place for them?

**Comment by igorkamyshev** [comment]
> > Sounds great 🤗
> > Shall we add some tests for this new store?
> 
> Oh, my bad, forgot about tests 😃 Is the `create_headless_query.test.ts` a right place for them?

Yeah, why not

File: apps/website/docs/api/primitives/query.md (hunk 2 above)
Comment (a-polunin) [reply apps/website/docs/api/primitives/query.md]: Added for a query and mutation :)

**Comment by igorkamyshev** [comment]
Great, thanks 🙏

**Pull request merged**